tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -135,3 +135,47 @@ describe('Test resource behavior after deleting GitRepo using keepResources opti
     }
   )
 });
+
+describe('Test Self-Healing of resource modification when correctDrift option used', { tags: '@p1'}, () => {
+  const repoName = "local-cluster-correct-drift"
+  const branch = "master"
+  const path = "qa-test-apps/nginx-app"
+  const repoUrl = "https://github.com/rancher/fleet-test-data/"
+  const appNamespace = 'nginx-keep'
+  const appName = 'nginx-keep'
+  const correctDriftData: testData[] = [
+    { qase_id: 76,
+      correctDrift: 'yes',
+      test_explanation: 'MODIFICATION to resources will be self-healed when correctDrift is set to true in GitRepo.',
+    },
+    { qase_id: 113,
+      correctDrift: 'no',
+      test_explanation: 'MODIFICATION to resources will not be self-healed when correctDrift is set to false in GitRepo.',
+    },
+  ]
+  correctDriftData.forEach(
+    ({ qase_id, correctDrift, test_explanation}) => {
+      qase(qase_id,
+        it(`Fleet-${qase_id}: Test ${test_explanation}`, { tags: `@fleet-${qase_id}` }, () => {
+          cy.fleetNamespaceToggle('fleet-local')
+          cy.addFleetGitRepo({ repoName, repoUrl, branch, path, correctDrift });
+          cy.clickButton('Create');
+          cy.checkGitRepoStatus(repoName, '1 / 1', '1 / 1');
+          cy.checkApplicationStatus(appNamespace, appName);
```

Comment:
Commented in [this](https://github.com/rancher/fleet-e2e/pull/87#discussion_r1546253405) other pr.
Locator on function `nameSpaceMenuToggle` (embedded within this one) needs to be updated at least for 2.7-head lane as it finds multiple ones.

Reply:
Okay, I will update there and here afterwards and see the difference.